js/scripts.js

```diff
@@ -1 +1,11 @@
-// target your button and assign an ask function that outputs a random image to your page
+
+var imgChange = document.getElementById("btn");
+var value = Math.ceil(Math.random() * 20);
+
+function eightBall() {
+  document.getElementById("magicImg").innerHTML = "<img src='../img/magic8ball_" + value + ".png'>";
+}
```

Comment:
you could also get your input by id and reset the value each time the eight ball function runs that way the user can ask a different question every time.
something like (`myInput.value = ''`) at the bottom of this function should do the trick